modules/deployment-scripts/import-acr/README.md

```diff
@@ -0,0 +1,83 @@
+# ACR Image Import
+
+An Azure CLI Deployment Script that imports public container images to an Azure Container Registry
+
+## Parameters
+
+| Name                 | Type     | Required | Description                                                                                                   |
+| :------------------- | :------: | :------: | :------------------------------------------------------------------------------------------------------------ |
+| `acrName`            | `string` | Yes      | The name of the Azure Container Registry                                                                      |
+| `location`           | `string` | No       | The location to deploy the resources to                                                                       |
+| `forceUpdateTag`     | `string` | No       | How the deployment script should be forced to execute                                                         |
+| `azCliVersion`       | `string` | No       | Version of the Azure CLI to use                                                                               |
+| `timeout`            | `string` | No       | Deployment Script timeout                                                                                     |
+| `retention`          | `string` | No       | The retention period for the deployment script                                                                |
+| `rbacRolesNeeded`    | `array`  | No       | An array of Azure RoleIds that are required for the DeploymentScript resource                                 |
+| `managedIdName`      | `string` | No       | Name of the Managed Identity resource                                                                         |
+| `images`             | `array`  | Yes      | An array of fully qualified images names to import                                                            |
+| `initialScriptDelay` | `string` | No       | A delay before the script import operation starts. Primarily to allow Azure AAD Role Assignments to propogate |
+| `cleanupPreference`  | `string` | No       | When the script resource is cleaned up                                                                        |
+
+## Outputs
+
+| Name | Type | Description |
+| :--- | :--: | :---------- |
+
+## Examples
+
+### Example 1
+
+```bicep
+param location string = resourceGroup().location
+param acrName string =  'yourAzureContainerRegistry'
+
+var imageName = 'ghcr.io/kedacore/keda-metrics-apiserver:main'
+
+module acrImport '../main.bicep' = {
```

Comment:
The examples should use the registry reference, not the local reference, so something like:

```bicep
module acrImport 'br/public:deployment-scripts/import-acr:1.0.1' = { ... }
```

@shenglol - do we have a reliable way to know what the initial version is going to be? Is at always `1.0.1`?

Reply:
Yes, it is always going to be `1.0.1`.

Reply:
updated